src/cdk/overlay/overlay-ref.ts

```diff
@@ -70,6 +67,19 @@ export class OverlayRef implements PortalOutlet {
     return this._host;
   }
 
+  /**
+   * Layout direction of the overlay panel.
+   */
+  get direction(): Direction {
```

Comment:
Could we make this a function `getDirection()` since `setDirection` is already there? I think that's more clear than having a separate readonly property

Reply:
Sure, done.